### ISSUE TITLE
React 17  bump update

### ISSUE THREAD

**Issue by leopucci** [sarneeh/reaptcha#219]
New versions on react 17 are not being able to use this code.

```
npm ERR! While resolving: pocketcloud@0.1.0  
npm ERR! Found: react@17.0.2  
npm ERR! node_modules/react  
npm ERR!   react@"^17.0.2" from the root project  
npm ERR!  
npm ERR! Could not resolve dependency:  
npm ERR! peer react@"^16.6.3" from reaptcha@1.7.2  
npm ERR! node_modules/reaptcha  
npm ERR!   reaptcha@"*" from the root project  
```

**Comment by leopucci** [comment]
Hi @sarneeh 
Please check it ok pull request to allow REACT 17 versions. 
https://github.com/sarneeh/reaptcha/pull/220

**Comment by sarneeh** [comment]
Thanks! Released in 1.7.3.

**Comment by leopucci** [comment]
Anytime 👍
